test/tests/jobs/CancelJobTest.cpp

```diff
@@ -84,18 +85,38 @@ struct CancelJobTest : tpunit::TestFixture {
         tester->executeWaitVerifyContent(command, "404 Invalid jobID - Cannot cancel a job with children");
     }
 
-    // Ignore canceljob for RUNNING jobs
+    // Ignore canceljob of RUNNING jobs
```

Comment:
Can you change this back to `for`?

Reply:
No idea, probably I wanted to write something else and wrote that when I undid it